edu.princeton.physics.WSJTX.metainfo.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>edu.princeton.physics.WSJTX.desktop</id>
+  <name>wsjtx</name>
+  <summary>Amateur Radio Weak Signal Operating</summary>
+  <developer_name>Joe Taylor and others</developer_name>
+  <description>
+  <p>
+WSJT-X Version 2.1 offers ten different protocols or modes: FT4, FT8,
```

Comment:
Here is a description from Debian that we could use:

> WSJT-X implements amateur radio communication protocols or "modes" called FT4, FT8, JT4, JT9, JT65, QRA64, ISCAT, MSK144, and WSPR, as well as one called Echo for detecting and measuring your own radio signals reflected from the Moon. These modes were all designed for making reliable, confirmed QSOs under extreme weak-signal conditions.
> 
> JT4, JT9, JT65, and QRA64 use nearly identical message structure and source encoding (the efficient compression of standard messages used for minimal QSOs). They use timed 60-second T/R sequences synchronized with UTC. JT65 and QRA64 were designed for EME ("moonbounce") on the VHF/UHF bands; JT65 has also proved popular and effective for worldwide QRP communication at HF. JT9 is optimized for the LF, MF, and HF bands. It is about 2 dB more sensitive than JT65 while using less than 10% of the bandwidth. With either JT9 or JT65, world-wide QSOs are possible with power levels of a few watts and compromise antennas. JT4 and QRA64 are optimized for EME on the VHF and higher bands, and especially the microwave bands from 2.3 to 24 GHz.
> 
> FT4 and FT8 are operationally similar but use T/R cycles only 7.5 and 15 s long, respectively. MSK144 is designed for Meteor Scatter on the VHF bands. These modes offer enhanced message formats with support for nonstandard callsigns and some popular contests.
> 
> WSPR mode implements a protocol designed for probing potential propagation paths with low-power transmissions. WSPR is fully implemented within WSJT-X, including programmable "band-hopping".

Reply:
Used the first paragraph